packages/react-test-renderer/npm/shallow.js

```diff
@@ -1,3 +1,8 @@
 'use strict';
 
-module.exports = require('react-shallow-renderer');
+var ReactShallowRenderer = require('react-shallow-renderer');
+
+// This is hacky but makes it work with webpack preferring to load ESM
+// even if imported via `require()`, and `react-shallow-renderer` ships with ESM.
+// See https://github.com/webpack/webpack/issues/7973
+module.exports = ReactShallowRenderer['default'] || ReactShallowRenderer;
```

Comment:
I've escaped `.default` as `['default']` intentionally (for super old setups 😅), but happy to make it look more modern.

Reply:
Ha, ESLint actually wants me to write `.default` anyway..